### DISCUSSION TITLE
Interval not respected and appName problem

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#79] [Q&A]
You can find my repository here: https://github.com/Cotrox/Repository-GitHub-ZeppOS-Apps

I'm trying to realize a sort of metronome App but how you can see the interval is not respected and the AppName is wrong. It show 'AppName' instead of his value.

How can i solve these problems?

**Comment by htoooth** [comment]
hi @Cotrox  

I have looked through your project, and it seems that the "i18n" file within the "gts4-mini" folder located under the "pages" directory is not functioning properly. It may be necessary to move this file directly under the "pages" folder in order for it to work properly.

**Reply by Cotrox** [reply]
How you can see in the updated repository i did it but the problem is still here :(

**Reply by htoooth** [reply]
@Cotrox  

 I make a PR to you.  https://github.com/Cotrox/Repository-GitHub-ZeppOS-Apps/pull/1

![image](https://user-images.githubusercontent.com/1717023/219616289-ce8e87e9-1658-4f75-9e40-2c89429749e8.png)

**Reply by Cotrox** [reply]
Hum! Thank you very much!

I was watching the App Store and it seams someone still my idea ahah someone realized something like this. Anyway, I will go on XD

Have you tested it? What do you think about time sync problem? Why vibrations interval is not respected?

**Reply by Cotrox** [reply]
It seams like if movements affect interval

**Reply by htoooth** [reply]
hi @Cotrox 
The Interval is not accurate, and you cannot rely on it. 
All timers run inside the rendering task, which can be delayed by the rendering task.

**Reply by Cotrox** [reply]
> hi @Cotrox 
> 
> The Interval is not accurate, and you cannot rely on it. 
> 
> All timers run inside the rendering task, which can be delayed by the rendering task.

Omg how can be possible?

**Reply by htoooth** [reply]
@Cotrox I'm sorry, but for now, you can only use the `interval` method.

**Reply by Cotrox** [reply]
Thank you! So is not possible to realize a Metronome App.
